funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -371,7 +371,8 @@ def stop_endpoint(self, name):
                 processes.append(parent)
                 for p in processes:
                     p.send_signal(signal.SIGTERM)
-                terminated, alive = psutil.wait_procs(processes, timeout=0.2)
+                # graceful RabbitMQ cleanup takes some time
+                terminated, alive = psutil.wait_procs(processes, timeout=2)
```

Comment:
This needed an increase because the RabbitMQ queue/channel cleanup seems to take a long time

Reply:
Hmm; just looking at the general surrounding code now.  IME, administrative signal-and-wait loops like this (as opposed to developer control tight-loops) typically have a longer grace period.  On the order of 10 to 30 seconds.  On a super busy system, perhaps with network delays, waiting up to 30 seconds in the outlier case would, I think, be "no problem."

Put differently, this script is nominally called either manually by a human, or during the host shutdown process, no?  In either case, given historical training (i.e., the UX presented by most other CLI tools), I'm fully prepared to wait a few seconds.  Given network and other delays, 30s is par for the course.

Good find.

Reply:
Yes, this is what happens when the user does `funcx-endpoint stop ...`

I increased it to 10 seconds, locally it generally takes around 1-2 seconds for me.